### PR TITLE
Remove experimental flag from `serverActions.bodySizeLimit`

### PR DESCRIPTION
**Pull request by devjiwonchoi**
Since the Server Actions became stable, the following config for `bodySizeLimit` should also be removed from the `experimental` section.

This PR added the `serverActions` option to the next config for the current `bodySizeLimit` and further additions.
Also, modified text on docs from `bytes` to `bits` and `MB` to `mb`.

Extends #57145 #57433

### CODE REVIEW

**Comment by ijjk** [comment]
Allow CI Workflow Run

- [ ] approve CI run for commit: 52719c2b4396e5c4808b68490134f150951e5b7c

Note: this should only be enabled once the PR is ready to go and can only be enabled by a maintainer

File: docs/02-app/02-api-reference/04-functions/server-actions.mdx [hunk 1]
```diff
@@ -140,16 +140,14 @@ In both cases, the form is interactive before hydration occurs. Although Server
 
 ## Size Limitation
 
-By default, the maximum size of the request body sent to a Server Action is 1MB, to prevent the consumption of excessive server resources in parsing large amounts of data.
+By default, the maximum size of the request body sent to a Server Action is `1mb`, to prevent the consumption of excessive server resources in parsing large amounts of data.
```
Comment (devjiwonchoi) [inline docs/02-app/02-api-reference/04-functions/server-actions.mdx:143]: https://github.com/vercel/next.js/blob/ccac12f69faabc1f52c7d558495ba75fb4cd2a7c/packages/next/src/server/app-render/action-handler.ts#L425

**Comment by devjiwonchoi** [comment]
Seems like #57529 is adding more options, need to resolve any conflicts if needed.

**Comment by devjiwonchoi** [comment]
@huozhi Got it. What do you think about the docs change though?

**Comment by huozhi** [comment]
@devjiwonchoi that sounds like an issue for configuration validation, thanks for reporting, we'll take a look on that!

**Comment by shuding** [comment]
Want to also add that even if Server Actions are considered stable, some options to tweak its behavior might still remain experimental (such as the body size thing). Mostly because they were introduced as a work around to certain issues that might be fixed in a different way in the future.

**Pull request closed without merging**